echo-bot.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3.10
+
+import logging
+import json
+from telegram import Update, ForceReply
+from telegram.ext import (
+    Updater,
+    CommandHandler,
+    MessageHandler,
+    CallbackContext,
+    Filters,
+)
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.DEBUG)
```

Comment:
In release it should be logging.INFO, I suppose

Reply:
Yes, of course, this is the initial version, so that we have something to start with.